### ISSUE TITLE
novelusb parser doesn't show all chapters

### ISSUE THREAD

**Issue by arromdee** [dteviot/WebToEpub#1037]
**Describe the bug**
Going to the site https://novelusb.com/novel-book/lord-of-the-mysteries-novel/ shows a novel with 1432 chapters.  This tool only sees the first 30.

**To Reproduce**
1. Go to 'https://novelusb.com/novel-book/lord-of-the-mysteries-novel/'
2. Run WebToEpub
3. See that it only shows 30 chapters.

**Expected behavior**
I expect to see 1432 chapters.

**Desktop (please complete the following information):**
 - OS: Mageia Linux 8
 - Browser Firefox 110.0 64 bit
 - Version  WebToEpub 0.0.0.155.  I had to use the prebuilt Firefox version from https://github.com/dteviot/WebToEpub since the official Firefox and Chrome stores only have 0.0.0.154 which doesn't support novelusb at all since it was added with https://github.com/dteviot/WebToEpub/issues/980

**Comment by SimplyBlue516** [comment]
I recommend you to click the chapter list. then wait for all the chapters to load then click webtoepub. I did run to a problem in which they don't load properly. I simply copy the URL then click apply. it loads all of the links. and I just edited the url chatpers cause they're a mess. and unselected some urls. plus. chapter 1 tends to be named "READ NOW" so I just edit the name in edit url chatpers. then cut and paste the last chapter to it's appropriate location. quite manual but at least can compile them.

**Comment by dteviot** [comment]
@SimplyBlue516 
Thanks for responding to arromdee.

>  I just edited the url chatpers cause they're a mess.

If you provide a URL that gives a messed up Table of Contents, I'll have a look and see what I can do.

**Comment by SimplyBlue516** [comment]
URL: https://novelbin.me/novel-book/signing-in-at-mount-sword-for-100-years-to-become-invincible#tab-chapters-title

First step: go to site and wait for all chapters to load.
2nd step: click webtoepub.
3rd step: click apply since it says that there's no parser found for this url.
4th step: you'll see the compilation and the mess it is. But I got no complaint since I just tried to fix it.
pretty much similar site with arromdee. So in my case, in default. If I compile. I'd pretty much have useless pages in my compilation and the chapters are all a mess due to the menu being included as well. got away with this by specifiying what to get on the urls but still some bits and stuff that is useless but overall neat enough to not mind. The thing is this started the day before yesterday and this isn't my first rodeo with novelbin/novelusb but before this I didn't have any problems in terms of compiling. So maybe they changed something.

**Comment by dteviot** [comment]
@SimplyBlue516 

I just got a patch from ImLJS that should fix novelbin.me.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.

**Comment by SimplyBlue516** [comment]
Okay, let me tell you my experience. About 2-3 hours before writing this(without reading your response). I went and packed another epub. and I had to do the manual labor of fixing the chapters again. But Just before checking github. I did another pack to a different novel but same site (novelbin). And it worked perfectly. Pretty much instant for packing no editing (other than Author and whatnot).

What I 'm saying is I looks likes it's good already but still experienced the problem a few hours ago. Don't know what happened  but should be good. I'll report if I see the problem again.
